ui/component/walletFiatAccountHistory/view.jsx

```diff
@@ -76,19 +61,19 @@ const WalletFiatAccountHistory = (props: Props) => {
                   />
                 </td>
                 <td>
-                  {getSymbol(transaction)}
-                  {transaction.tipped_amount / 100} {getCurrencyIso(transaction)}
+                  {STRIPE.CURRENCY[transaction.currency.toUpperCase()].symbol}
+                  {transaction.tipped_amount / 100} {STRIPE.CURRENCIES[transaction.currency.toUpperCase()]}
                 </td>
                 <td>
-                  {getSymbol(transaction)}
+                  {STRIPE.CURRENCY[transaction.currency.toUpperCase()].symbol}
                   {transaction.transaction_fee / 100}
                 </td>
                 <td>
-                  {getSymbol(transaction)}
+                  {STRIPE.CURRENCY[transaction.currency.toUpperCase()].symbol}
                   {transaction.application_fee / 100}
                 </td>
                 <td>
-                  {getSymbol(transaction)}
+                  {STRIPE.CURRENCY[transaction.currency.toUpperCase()].symbol}
```

Comment:
:grey_question: This will crash if the returned transaction currency is, for whatever reason, not USD or EUR.